build.gradle

```diff
@@ -64,21 +66,42 @@ allprojects {
   apply plugin: 'com.github.spotbugs'
 
   dependencies {
-    spotbugsPlugins 'com.h3xstream.findsecbugs:findsecbugs-plugin:1.10.1'
+    spotbugsPlugins 'com.h3xstream.findsecbugs:findsecbugs-plugin:1.11.0'
   }
 
   spotbugs {
-    toolVersion = '4.0.3'
-    sourceSets = [ sourceSets.main ]
+    toolVersion = '4.2.0'
     excludeFilter = file("${rootProject.projectDir}/findbugs-supressions.xml")
   }
 
-  // To generate an HTML report instead of XML
-  tasks.withType(SpotBugsTask) {
-    reports.xml.enabled = false
-    reports.html.enabled = true
+  spotbugsMain {
+    reports(({
+      html {
+        enabled = true
+        destination = file("$buildDir/reports/spotbugs/main/spotbugs.html")
+        stylesheet = 'fancy-hist.xsl'
+      }
+      xml {
+        enabled = false
+      }
+    } as Closure<NamedDomainObjectContainer<? extends SpotBugsReport>>))
   }
 
+  spotbugsTest {
```

Comment:
Does this look for bugs in test files? Or is this a test of SpotBugs?

Reply:
Bugs in the Test files